<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagEdition</class>
 <widget class="QDialog" name="TagEdition">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tag Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLineEdit" name="uriTextedit"/>
     </item>
     <item>
      <widget class="QPushButton" name="fileBrowser">
       <property name="text">
        <string>Browser</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titlelabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>TitleEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="TitleEdit"/>
     </item>
     <item row="0" column="5">
      <widget class="QCheckBox" name="cbtitre">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="artistlabel">
       <property name="text">
        <string>Artist:</string>
       </property>
       <property name="buddy">
        <cstring>artistEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QLineEdit" name="artistEdit"/>
     </item>
     <item row="1" column="5">
      <widget class="QCheckBox" name="cbartist">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="albumlabel">
       <property name="text">
        <string>Album:</string>
       </property>
       <property name="buddy">
        <cstring>albumEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QLineEdit" name="albumEdit"/>
     </item>
     <item row="2" column="5">
      <widget class="QCheckBox" name="cbalbum">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="trackLabel">
       <property name="text">
        <string>Track number:</string>
       </property>
       <property name="buddy">
        <cstring>trackEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QLineEdit" name="trackEdit">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QCheckBox" name="cbTrack">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="durationlabel">
       <property name="text">
        <string>Duration:</string>
       </property>
       <property name="buddy">
        <cstring>timeEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTimeEdit" name="timeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="3">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>177</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelYear">
       <property name="text">
        <string>Year:</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="5" column="3" colspan="2">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="5">
      <widget class="QCheckBox" name="cbYear">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="commentlabel">
       <property name="text">
        <string>Comment:</string>
       </property>
       <property name="buddy">
        <cstring>commetEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="4">
      <widget class="QLineEdit" name="commetEdit"/>
     </item>
     <item row="6" column="5">
      <widget class="QCheckBox" name="cbcomment">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="kindlabel">
       <property name="text">
        <string>Genre:</string>
       </property>
       <property name="buddy">
        <cstring>genreCombobox</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QComboBox" name="genreCombobox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="5">
      <widget class="QCheckBox" name="cbgenre">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>211</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelbutton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uriTextedit</tabstop>
  <tabstop>fileBrowser</tabstop>
  <tabstop>TitleEdit</tabstop>
  <tabstop>cbtitre</tabstop>
  <tabstop>artistEdit</tabstop>
  <tabstop>cbartist</tabstop>
  <tabstop>albumEdit</tabstop>
  <tabstop>cbalbum</tabstop>
  <tabstop>trackEdit</tabstop>
  <tabstop>cbTrack</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>cbYear</tabstop>
  <tabstop>commetEdit</tabstop>
  <tabstop>cbcomment</tabstop>
  <tabstop>genreCombobox</tabstop>
  <tabstop>cbgenre</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelbutton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
